### PR TITLE
Config with zope product template 18ffe3f7

### PR DESCRIPTION
**Pull request by icemac**

### CODE REVIEW

**Comment by icemac** [comment]
Got GHA green, ready to review.

File: CHANGES.rst [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
-0.9.1 (unreleased)
-==================
+1.0 (unreleased)
+================
+
+* Drop support for Python 2.7, 3.5, 3.6.
+
+* Add support for Python 3.9, 3.10, 3.11.
+
+* Drop support for Python 2.7, 3.5, 3.6.
```
Comment (gforcada) [inline CHANGES.rst]: Somehow this line is duplicated just above :)

Comment (mauritsvanrees) [reply CHANGES.rst]: And it is above the sub headers.

**Review by mauritsvanrees — APPROVED** [review]
LGTM, thanks, except a minor thing in the changelog.

**Comment by icemac** [comment]
Thank you for reviewing this PR. 😃

**Pull request merged**